plugins/snowflake/scripts/classify_issues.py

```
@@ -303,6 +312,14 @@ def main():
         pct = count / len(output) * 100
         print(f"  {cat:<45s} {count:>4d} ({pct:.1f}%)")
 
+    # Print bot/human split
+    bot_count = sum(1 for item in output if item.get("is_bot"))
+    human_count = len(output) - bot_count
+    if bot_count > 0:
+        print(f"\nBot/Human Split:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)